contents/blog/posthog-vs-pendo.md

```diff
@@ -0,0 +1,652 @@
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Write your own queries in SQL</td>
+        </tr>
+    </tbody>
+</table>
+</div>
+
+- **[Correlation analysis](/product-analytics):** Correlation analysis is a feature which enables you to automatically discover significant linking properties or events relevant to an insight. For example, you can find if users who complete a funnel are likely to be from a certain location, or have completed another event. In PostHog, correlation analysis is a standard part of analytics. 
+
+- **[Formulas](/product-analytics):** In PostHog, a formula is exactly what it sounds like — a mathematical process you can use as part of your analysis. A simple example of a formula would be an equation to figure out a ratio or percentage, though advanced formulas can use more elaborate functions, such as `COS` and `SIN`. Pendo, however, doesn’t offer the ability to use formulas in analysis.
+
+- **[SQL access](/docs/product-analytics/hogql):** SQL, which stands for Structured Query Language, is a programming language for interrogating data. While both Pendo and PostHog offer the ability to analyze data with ready-made insights and visualizations types, SQL enables you to go beyond this and examine data in any way. Only PostHog offers SQL access to your data; in Pendo you are limited to the existing graphs and widgets.
+
+<h4 className="mb-0">Discover what's possible with <span className="text-red">product analytics</span></h4>
+<TutorialsSlider slugs={[
+  "/tutorials/churn-rate",
+  "/tutorials/funnels",
+  "/tutorials/api-get-insights-persons",
+  "/tutorials/feature-retention",
+  "/tutorials/next-steps-after-installing"
+]} />
+
+### In-app messages and prompts
+The ability to send in-app messages and display prompts is a central feature of Pendo, which focuses on empowering users to create such pop-ups and tooltips in their product easily. Additionally, you can see how adding such prompts impacts adoption and retention, or display interactive prompts to solicit user feedback. 
+
+Just because such features are central to Pendo doesn’t mean they’re unique, however. PostHog also offers the ability to display in-app messages and interactive pop-ups through the use of [feature flag payloads](/docs/feature-flags/payloads) and [site apps](/tutorials/build-site-app).
+
+The main difference between PostHog and Pendo here is that, because Pendo is intended for less technical users, it offers an interface which is simpler, but less powerful. You can easily create tooltips, announcement pop-ups and polls in Pendo, but not other types of prompt such as a Calendly integration. In PostHog, there are ready made apps for displaying pop-ups and tooltips, but the true power is in the ability to [build custom solutions](/tutorials/build-site-app).
```

Comment:
```suggestion
Pendo is all about creating in-app messages, prompts, tooltips, measuring their impact on adoption and retention, and collecting feedback. But these aren't unique features – PostHog also supports in-app messages and interactive pop-ups through the use of [feature flag payloads](/docs/feature-flags/payloads) and [site apps](/tutorials/build-site-app).

The main difference between is, because Pendo is designed for less technical users, it offers an interface that is simpler, but less powerful. You can easily create tooltips, announcement pop-ups and polls in Pendo, but not other types of prompt such as a Calendly integration. 

PostHog has ready-made apps for displaying pop-ups and tooltips, but the true power is in the ability to [build custom solutions](/tutorials/build-site-app).
```